src/App.css

```diff
@@ -232,3 +232,28 @@ select option {
 .button .btn {
   text-transform: uppercase;
 }
+
```

Comment:
- [ ] [OPTIONAL] 
The _categories_ input field around the form section is missing! Let's ensure to include it, and also maintain the _title_ and _author_ input fields, all 3 inputs on one line.

  **Kindly include the categories section:**
![image](https://github.com/ybadr99/bookstore/assets/12745474/74b9cdc6-c305-456c-8ab8-a2c723af9e58)